src/wp-admin/includes/plugin-install.php

```diff
@@ -634,7 +634,7 @@ function install_plugin_information() {
 				) );
 				?>
 				<div class="counter-container">
-						<span class="counter-label"><a href="https://wordpress.org/support/plugin/<?php echo $api->slug; ?>/reviews/?filter=<?php echo $key; ?>"
+						<span class="counter-label"><a href="https://forums.classicpress.net/plugin/<?php echo $api->slug; ?>/reviews/?filter=<?php echo $key; ?>"
```

Comment:
This change is incorrect as we do not have subforums for each plugin.

Reply:
Let's leave this one pointing to WP too.